code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -16,6 +16,7 @@
 	var/effectmod
 	var/list/activate_reagents = list() //Reagents required for activation
 	var/recurring = FALSE
+	var/research //Research point value
```

Comment:
If it defaults to none you should probably check if it has any value when trying to redeem it.